rocAL/rocAL_pybind/example/new_api/parse_config.py

```diff
@@ -36,14 +11,17 @@ def parse_args():
                         help='image folder files')
     common_group.add_argument('--batch-size', '-b', type=int, default=10,
                         help='number of examples for each iteration')
-    add_bool_arg(common_group, 'display', default=True)
-    add_bool_arg(common_group, 'print_tensor', default=True)
-    add_bool_arg(common_group, 'classification', default=True)  # use --classification for Classification / --no-classification for Detection
-    add_bool_arg(common_group,'rocal-gpu', default=True)
-
-    common_group.add_argument('--NHWC', action='store_true',
+    common_group.add_argument('--display', default=False, action="store_true",
+                        help='--display:to display output from the pipeline')
+    common_group.add_argument('--print_tensor', default=False, action="store_true",
+                        help='--print_tensor: to print tensor output from the pipeline')
+    common_group.add_argument('--classification', default=False, action="store_true",
+                        help='--classification: to use for classification')
+    common_group.add_argument('--rocal-gpu', default=False, action="store_true",
```

Comment:
@rrawther Seems like this should default to "True" as per line 42 in old code

Reply:
@LakshmiKumar23: I fixed the changes for this. Please recheck